samples/bluetooth/broadcast_audio_sink/Kconfig

```diff
@@ -43,4 +43,35 @@ config ENABLE_LC3
 	select LIBLC3
 	select FPU
 
+config USE_USB_AUDIO_OUTPUT
+	bool "Use USB Audio as output"
+	# By default, use the USB Audio path is disabled.
+	default n
+	depends on ENABLE_LC3
+	select USB_DEVICE_STACK
+	select USB_DEVICE_AUDIO
+	select RING_BUFFER
+	help
+	  Enables USB audio as output as a USB peripheral. This does not support providing USB
+	  audio to e.g. speakers that are also USB peripherals, but can be connected to e.g. a
+	  phone or PC as a USB-in device (such as a USB microphone).
+	  USB audio only supports a single audio channel.
+
+config USE_AUTOMATIC_BROADCAST_SYNC
+	bool "Automatic Broadcast Sync"
+	default y
+	depends on USE_USB_AUDIO_OUTPUT
+	help
+	  Instead of using a broadcast assistant, sync automatically to first broadcast which
+	  has the configured TARGET_BROADCAST_CHANNEL.
+
+config TARGET_BROADCAST_CHANNEL
+	int "Broadcast Channel Audio Location to sync to"
+	range 0 2
+	default 1
+	depends on USE_USB_AUDIO_OUTPUT
+	help
+	  Channel Audio Location to sync to. These corresponds to the bt_audio_location,
+	  supporting mono, left and right channels
```

Comment:
Should this be optional? i.e. if not set, sync to the first BIS?

Reply:
I think currently this can make sense. We are acting as either left or right side (or mono). If the right channel does not exist, then there isn´t a stream we are interested in.

I understand where you come from, but I would prefer that we add this in an upcoming PR, along with possible stereo support.